src/consumer.ts

```diff
@@ -134,6 +139,31 @@ export class Consumer extends TypedEventEmitter {
     return !this.stopped;
   }
 
+  /**
+   * Updates the provided option to the provided value.
+   * @param option The option that you want to update
+   * @param value The value to set the option to
+   */
+  public updateOption(
+    option: UpdatableOptions,
+    value: ConsumerOptions[UpdatableOptions]
+  ) {
+    switch (option) {
+      case 'visibilityTimeout':
+        if (typeof value === 'number') {
+          debug(`Updating the visibilityTimeout option to the value ${value}`);
+
+          this.visibilityTimeout = value;
+
+          this.emit('option_updated', option, value);
+        }
+
+        break;
+      default:
+        break;
```

Comment:
Possibly need to add something here for if the option sent is not updatable, although I'm in two minds as to if this is actually needed, as it's typed.